examples/user_interfaces/embedding_in_wx4_sgskip.py

```diff
@@ -21,29 +21,21 @@ class MyNavigationToolbar(NavigationToolbar):
 
     def __init__(self, canvas, cankill):
         NavigationToolbar.__init__(self, canvas)
-
         # for simplicity I'm going to reuse a bitmap from wx, you'll
         # probably want to add your own.
         tool = self.AddTool(wx.ID_ANY, 'Click me', _load_bitmap('back.png'),
                             'Activate custom contol')
         self.Bind(wx.EVT_TOOL, self._on_custom, id=tool.GetId())
 
     def _on_custom(self, evt):
-        # add some text to the axes in a random location in axes (0,1)
-        # coords) with a random color
-
-        # get the axes
-        ax = self.canvas.figure.axes[0]
-
-        # generate a random location can color
-        x, y = np.random.rand(2)
-        rgb = np.random.rand(3)
-
-        # add the text and draw
-        ax.text(x, y, 'You clicked me',
-                transform=ax.transAxes,
-                color=rgb)
-        self.canvas.draw()
+        # add some text to the axes in a random location in axes coords with a
+        # random color
+        ax = self.canvas.figure.axes[0]  # get the axes
+        x, y = np.random.rand(2)  # generate a random location
+        rgb = np.random.rand(3)  # generate a random color
+        ax.text(  # add the text
+            x, y, 'You clicked me', transform=ax.transAxes, color=rgb)
+        self.canvas.draw()  # draw
```

Comment:
```suggestion
        self.canvas.draw()
```
No information, so just leave out. The `# add the text` and `# get the axes` are almost as meaningless and could as well be removed.

Reply:
fair enough, removed all of them